oum.html.markdown

```diff
@@ -0,0 +1,181 @@
+---
+  language: Oum
+  contributors:
+    - ["Theo Butler", "https://github.com/Theodus"]
+  filename: learnoum.oum
+---
+
+Oum was created to provide a scripting language which could be implemented
+within Java projects or as a stand-alone language with the possibility of sharing
+variables and methods with Java classes.
+
+The GitHub repository for the project may be found at (https://github.com/Theodus/Oum)
+You can give feedback to TheodusButler [at] [google's email service]
+
+```
+
+// Comments start with two forward slashes, same as Java
+
+///////////////////////////
+//  Objects & Operators  //
+///////////////////////////
+
+// all values are objects and can be created as objects
+
+a = 3
+// or
+a = new OumInteger(3)
+
+// basic math
+
+1 + 1 //=> 2
+2 - 1 //=> 1
+2 * 2 //=> 4
+
+// exponent returns a double
+
+2 ^ 3 //=> 8.0
+
+// division of integers returns an integer
+
+5 / 2  //=> 2
+-5 / 2 //=> -2
+
+// if at least one of the nubers is a double value a double will be returned
+
+5.0 / 2 //=> 2.5
+
+// modulus operation returns the remainder of a division 
```

Comment:
you're sometimes using "exponent" and sometimes "modulus **operation**". any reason?